### ISSUE TITLE
Why the v2xvit result of  ap_50 and ap_70 is lower than paper result?

### ISSUE THREAD

**Issue by HuangZhe885** [DerrickXuNu/OpenCOOD#54]

**Comment by HuangZhe885** [comment]
I use the given pre-training v2xvit model to test directly
The result is :
ap30: 0.8923328503252944
ap_50: 0.8408056739993565
ap_70: 0.6163396643250969

But the results in the paper are much higher than that. How can I adjust the model to the same results as the paper?
The version of spconv is 2.0

**Comment by DerrickXuNu** [comment]
The provided model's performance is the same and even higher as the paper claim. The V2X-ViT has AP0.5/0.7 for 0.836/0.614 under noisy setting, in the origin paper. I assume you are talking about the perfect setting result? If that is the case, you need to change the async and gps_error to false in the yaml file to test on perfect setting

**Comment by HuangZhe885** [comment]
Thank you for your reply, I measured it under the condition of no noise，the yaml file is
`wild_setting:
  async: false
  async_mode: 'sim'
  async_overhead: 100
  seed: 25
  loc_err: false
  xyz_std: 0.2
  ryp_std: 0.2
  data_size: 1.06 # Mb!!
  transmission_speed: 27 # Mbps!!
  backbone_delay: 10 # ms`
the result is AP0.5/0.7 for 0.84/0.62, but the perfect result is 88.2/71.2
I didn't change any configuration of original yaml file

**Comment by HuangZhe885** [comment]
> The provided model's performance is the same and even higher as the paper claim. The V2X-ViT has AP0.5/0.7 for 0.836/0.614 under noisy setting, in the origin paper. I assume you are talking about the perfect setting result? If that is the case, you need to change the async and gps_error to false in the yaml file to test on perfect setting

Thank you for your reply, I measured it under the condition of no noise，the yaml file is
`wild_setting: async: false async_mode: 'sim' async_overhead: 100 seed: 25 loc_err: false xyz_std: 0.2 ryp_std: 0.2 data_size: 1.06 # Mb!! transmission_speed: 27 # Mbps!! backbone_delay: 10 # ms`
the result is AP0.5/0.7 for 0.84/0.62, but the perfect result is 88.2/71.2
I didn't change any configuration of original yaml file

**Comment by DerrickXuNu** [comment]
The pretrained model that I released has the exact same number as the paper, please carefully check it. As for your training, don't use the last epoch as your final model as it may overfit. Use the validation curve to find the best one. Also, some users have already easily get higher score, check this: https://github.com/DerrickXuNu/v2x-vit/issues/7

**Comment by HuangZhe885** [comment]
> The pretrained model that I released has the exact same number as the paper, please carefully check it. As for your training, don't use the last epoch as your final model as it may overfit. Use the validation curve to find the best one. Also, some users have already easily get higher score, check this: [DerrickXuNu/v2x-vit#7](https://github.com/DerrickXuNu/v2x-vit/issues/7)

Thanks again for your suggestions, replies  and  your excellent works! This is already my study material because it is really beginning friendly.

**Comment by DerrickXuNu** [comment]
Glad to hear that! I will close the comment for now and let me know if you have further questions
